APIs/schemas/receiver_audio.json

```
@@ -25,7 +25,7 @@
           "type": "object",
           "properties": {
             "media_types": {
-              "description": "Subclassification of the formats accepted using IANA assigned media types",
+              "description": "Subclassification of the formats accepted using IANA assigned media types, if available",
```

Should we try to write it in such a way so it doesn't seem that the field is optional if there isn't an IANA assigned media type?

```suggestion
              "description": "Subclassification of the formats (prefer IANA assigned media types where available)",
```